src/js/outreach/views/opt-in_views.js

```diff
@@ -53,20 +72,35 @@ const OptInView = View.extend({
     <p class="opt-in__disclaimer">By clicking Submit you agree to receive SMS text message notifications. You may opt out at any time.</p>
     <button class="opt-in__submit button--green w-100 js-submit" disabled>Submit</button>
   `,
+  templateContext() {
+    const dobMonthRange = range(1, 13);
+    const dobDayRange = range(1, 32);
+    const dobYearRange = range(1900, 2015).reverse();
```

Comment:
Did this so we can do `{{#each arrayOfValues}}`'s in the handlebars template. 

Probably want to do this different somehow.

Reply:
Also not sure what year range we want to use. And if we want to display the month in a numerical or name format (1 vs. "Jan")

Reply:
I suspect the year range should be chosen programatically..